backend/canisters/group/api/src/updates/convert_into_community.rs

```diff
@@ -6,6 +6,7 @@ use types::{AccessRules, CommunityId, CommunityPermissions};
 pub struct Args {
     pub rules: AccessRules,
     pub permissions: Option<CommunityPermissions>,
+    pub primary_language: Option<String>,
```

Comment:
Should this be optional? I feel like it should be mandatory...

Reply:
Yeah I suppose so